packages/Upgrade/src/api/repositories/UserRepository.ts

```diff
@@ -1,5 +1,69 @@
+import 'reflect-metadata';
```

Comment:
Do we need this. We have already imported this in app.ts

Reply:
Oh no actually when i tried to use systemUserDoc directly in where clause at that time it was throwing error of reflect metadata. and hence imported this

Removed this now